apps/coscrad-frontend-e2e/src/e2e/resources/videos/video.index.query-flow.e2e.cy.ts

```diff
@@ -0,0 +1,204 @@
+                            .click()
+                            .get(`[data-value="${searchScope}"]`)
+                            .click();
+                    });
+
+                    describe(`when the filter should return 1 result (based on title)`, () => {
+                        it(`should return the correct result`, () => {
+                            const searchTerms = `Q-`;
+
+                            cy.getByDataAttribute(`index_search_bar`).click().type(searchTerms);
+
+                            cy.getLoading().should(`not.exist`);
+
+                            cy.contains(videoTitleWithQDash);
+
+                            cy.contains(dummyVideoTitle).should('not.exist');
+                        });
+                    });
+
+                    describe(`when the filter should return 1 result (based on titleEnglish)`, () => {
+                        it(`should return the correct result`, () => {
+                            const searchTerms = `ZZZ`;
+
+                            cy.getByDataAttribute(`index_search_bar`).click().type(searchTerms);
+
+                            cy.getLoading().should(`not.exist`);
+
+                            cy.contains(dummyVideoTitle);
+
+                            cy.contains(videoTitleWithQDash).should('not.exist');
+                        });
+                    });
+
+                    describe(`when the filter should return no results`, () => {
+                        it(`should show no results`, () => {
+                            const searchTerms = `BBQ Chicken`;
+
+                            cy.getByDataAttribute(`index_search_bar`).click().type(searchTerms);
+
+                            cy.getLoading().should(`not.exist`);
+
+                            cy.contains(dummyVideoTitle).should(`not.exist`);
+
+                            cy.contains(videoTitleWithQDash).should('not.exist');
+
+                            cy.getByDataAttribute(`not-found`);
+                        });
+                    });
+
+                    describe(`when the filter should return all results`, () => {
+                        it(`should show all results`, () => {
+                            const searchTerms = `n`;
+
+                            cy.getByDataAttribute(`index_search_bar`).click().type(searchTerms);
+
+                            cy.getLoading().should(`not.exist`);
+
+                            cy.contains(dummyVideoTitle);
+
+                            cy.contains(videoTitleWithQDash);
+
+                            cy.getByDataAttribute(`not-found`).should('not.exist');
+                        });
+                    });
+
+                    /**
+                     * For some strange reason, cy.contains seems to completely
+                     * fail with the following special characters. The screenshot
+                     * clearly shows the character on the screen, and yet it is
+                     * not found. Further,
```

Comment:
This is my comment. Not sure why it trails off. Please remove `Further,` for now.